_posts/2022-02-18-partner-highlight-titaniam-byok.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: post
+title:  "OpenSearch Partner Highlight: BYOK for B2B SaaS Operators using OpenSearch"
+authors:
+  - pakshirajan 
+date:   2022-02-18 01:01:01 -0700
```

Comment:
Hello @pakshi - can you update the date on this? While you're at it, please also update the filename to match the more recent date "2022-**02-18**-partner-highlight-titaniam-byok.md" 
thank you

Reply:
@krisfreedain I made the changes